runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtOverlay.java

```diff
@@ -53,7 +53,7 @@ private WintertodtOverlay(WintertodtPlugin plugin)
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		if (!plugin.isInWintertodt())
+		if (!plugin.isInWintertodt() || plugin.clientOverlayToggle == false)
```

Comment:
`boolean == false` is the same as `!boolean`, use the latter form.

Also, don't store the config state in plugin, just inject the config and directly call the config method here instead.